src/transcript/TranscriptEntity.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export default class TranscriptEntity {
```

Comment:
This whole thing needs a block comment and an example, and you probably need more restricted types than `string`. 

`endTimeMs` seems to be a Unix timestamp in milliseconds. 

`type` seems like it can be `null`, so you need ` | null` in there. It also seems to be a restricted set — maybe

```
'PUNCTUATION' | 'Address' | … | null
```

right?

Reply:
Anything that's an _input_ to Transcribe can be a restricted set; we can add to it so long as we document that it can grow.

Anything that's _returned_ from Transcribe and can change in the future needs to be an open value, and also needs to be heavily documented that it can change as Transcribe evolves!

Reply:
Added extra documentation for entity types explaining different types that we support now.